openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2033,6 +2033,10 @@ The following is a non-normative example of a Credential Response with Credentia
 
 <{{examples/credential_response_jwt_vc_json.txt}}
 
+The following is a non-normative example of a dereferenced document for the Issuer HTTP URL identifier.
```

Comment:
Maybe something like this if I've understood what is meant correctly:

```suggestion
The following is the dereferenced document for the Issuer HTTP URL identifier that matches the Credential in the above example:
```

Reply:
Thanks